src/main/java/nl/hsac/fitnesse/fixture/slim/JsonFixture.java

```diff
@@ -73,7 +73,7 @@ public String createFileFromBase64ContentOf(String baseName, String jsonPath) {
 
     public Object elementOfJsonPath(int index, String path) {
         List<Object> all = listJsonPathMatches(path);
-        return all.get(index);
+        return all.size() > 0 ? all.get(index) : null;
```

Comment:
```suggestion
        return all.size() > index ? all.get(index) : null;
```

Reply:
Ah, yes... Interesting. This would make sure that when you chose an element which does not exist.